src/main/java/appeng/items/parts/FacadeItem.java

```diff
@@ -79,6 +82,12 @@ public InteractionResult onItemUseFirst(ItemStack stack, UseOnContext context) {
             }
         }
 
+        BlockPos blockPos = context.getClickedPos();
+        Minecraft minecraft = Minecraft.getInstance();
+        BlockState blockState = this.getTextureBlockState(stack);
+        SoundType soundType = blockState.getSoundType();
+
+        minecraft.level.playLocalSound(blockPos, soundType.getPlaceSound(), SoundSource.BLOCKS, 1.0f, 1.0f, false);
```

Comment:
This is not the right place, this should really fire in the code where the facade is placed.

Reply:
I think inside `placeFacade` makes more sense, after the call to `addFacade`.

Reply:
Thanks, that code has been moved to placeFacade.